src/command_modules/azure-cli-recoveryservices/HISTORY.rst

```diff
@@ -0,0 +1,9 @@
+.. :changelog:
+
+Release History
+===============
+
+0.0.1b1 (YYYY-MM-DD)
```

Comment:
Just put "unreleased" here and Derek will assign the appropriate stuff when the release occurs.